.circleci/publish_to_github.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# TODO(josephperrott): Remove this script in favor of common orb to perform the task.
```

Comment:
I assume something like this could also be a nice script as part of the shared dev-infra package (written in JS)?

Reply:
Yeah, thats the general idea, I am not sure of the exact implementation, but the overall idea sounds correct to me.